### ISSUE TITLE
I did the portuguese BR translation

### ISSUE THREAD

**Issue by an0nimo** [valinet/ExplorerPatcher-L10N#26]
Thank you so much for your great job with **ExplorerPatcher** and my small contribution that I can do is the translation for portuguese BR. I used the files ep_gui.en-US.rc and ep_setup.en-US.rc like base to do that in notepad++ program.
The files are attached to be checking.
Thanks.
[pt-BR.zip](https://github.com/valinet/ExplorerPatcher-L10N/files/13625383/pt-BR.zip)

**Comment by Amrsatrio** [comment]
Hi, can you try to perform a pull request instead of this so that you will be properly credited?

**Comment by an0nimo** [comment]
I've tried many times but always is shown this message:
"_Choose different branches or forks above to discuss and review changes. [Learn about pull requests](https://docs.github.com/articles/using-pull-requests)_"
For that, I create the post here and about the credits, I don't care about that.

**Comment by NicoBR333** [comment]
> I've tried many times but always is shown this message: "_Choose different branches or forks above to discuss and review changes. [Learn about pull requests](https://docs.github.com/articles/using-pull-requests)_" For that, I create the post here and about the credits, I don't care about that.

Oh, nice; I was about to do a Portuguese-BR translation; and I actually got some progress at 6th of December, but for some reason it still haven't been added.

Well, I'm gonna check for a bit and I'm gonna reply back later on

**Comment by NicoBR333** [comment]
Hm, right here you can put "Menu do Windows 7", since it doesn't really have much
![image](https://github.com/valinet/ExplorerPatcher-L10N/assets/77538136/23b676e5-3ad0-4d53-bff2-0ce6495ff2ba)
![image](https://github.com/valinet/ExplorerPatcher-L10N/assets/77538136/07d83ee9-9f1a-4612-95c3-5fe4535b6275)

**Comment by NicoBR333** [comment]
Also, can I help adapt your translation?

**Comment by Amrsatrio** [comment]
Of course. But tossing files each time a change is made, I think, isn't a convenient and reliable way for this.

**Comment by NicoBR333** [comment]
> Of course. But tossing files each time a change is made, I think, isn't a convenient and reliable way for this.

Yeah, that makes sense; probably it's a better idea to have a conclusion from @an0nimo to decide when will it be the definitive and best translation (maybe this will end up being editted over 5 times and probably a week).

After it got concluded, I would have to upload it with the credit saying that 99% of the stuff has been made by him (not even doing much)

**Comment by an0nimo** [comment]
I used ChatGPT to translate the bulk.
So I started reviewing it sentence by sentence to fine-tune the correctness, but as I don't know all the menus, so there are things that seemed correct to me, but in the context it needs to adapt more to the way of speaking the language.

**Comment by NicoBR333** [comment]
> I used ChatGPT to translate the bulk. So I started reviewing it sentence by sentence to fine-tune the correctness, but as I don't know all the menus, so there are things that seemed correct to me, but in the context it needs to adapt more to the way of speaking the language.

Oh, yeah, I think I should've have tried to do that before, but I was doing it from scratch so it wasn't really the most efficient lol

I've got to do some changes, but what's the best?
# #1 -
    IDS_EXP_CTRLINTF_0_W11      "Command Bar do Windows 11 (padrão)"
    IDS_EXP_CTRLINTF_0_W10      "Ribbon do Windows 10 (padrão)"
    IDS_EXP_CTRLINTF_1_W11      "Ribbon do Windows 10"
    IDS_EXP_CTRLINTF_2          "Command Bar do Windows 7"
    IDS_EXP_CTRLINTF_3          "Command Bar do Windows 11 (sem guias, Barra de Endereços legada)"
    IDS_EXP_CTRLINTF_4          "Command Bar do Windows 11 (Barra de Endereços legada)"

# #2 -
    IDS_EXP_CTRLINTF_0_W11      "Barra de Comando do Windows 11 (padrão)"
    IDS_EXP_CTRLINTF_0_W10      "Faixa do Windows 10 (padrão)"
    IDS_EXP_CTRLINTF_1_W11      "Faixa do Windows 10"
    IDS_EXP_CTRLINTF_2          "Barra de Comando do Windows 7"
    IDS_EXP_CTRLINTF_3          "Barra de Comando do Windows 11 (sem guias, Barra de Endereços legada)"
    IDS_EXP_CTRLINTF_4          "Barra de Comando do Windows 11 (Barra de Endereços legada)"

**Comment by NicoBR333** [comment]
Made some adaptations but not all of them. What do you think?
[ep_PT-BR.zip](https://github.com/valinet/ExplorerPatcher-L10N/files/13654381/ep_PT-BR.zip)
